site/quickstart.md

```diff
@@ -133,15 +133,15 @@ $ cd kubernetes/samples/scripts/create-weblogic-domain-credentials
 $ ./create-weblogic-credentials.sh -u weblogic -p welcome1 -n sample-domain-ns1 -d sample-domain1
 ```
 
-The sample will create a secret named `domdinUID-weblogic-credentials` where the `domainUID` is replaced 
+The sample will create a secret named `domdinUID-weblogic-credentials` where the `domainUID` is replaced
```

Comment:
would you mind fixing the typo in "domainUID" please

Reply:
No problem. Done. (Sorry I missed it.)